pkg/porter/run.go

```diff
@@ -238,32 +242,9 @@ func (p *Porter) ApplyBundleOutputs(opts RunOptions, outputs map[string]string)
 				}
 
 				if doApply {
-					// Ensure outputs directory exists
-					if err := p.FileSystem.MkdirAll(config.BundleOutputsDir, 0755); err != nil {
-						return errors.Wrap(err, "unable to ensure CNAB outputs directory exists")
-					}
-
 					outpath := filepath.Join(config.BundleOutputsDir, bundleOutput.Name)
 
-					outputType, ok, err := bundleOutput.Schema.GetType()
-					if !ok && err != nil {
-						return errors.Wrap(err, "unable to get output type")
-					}
-
-					// Create data structure with relevant data for use in listing/showing later
-					output := config.Output{
-						Name:      bundleOutput.Name,
-						Sensitive: bundleOutput.Sensitive,
-						Type:      outputType,
-						Value:     outputValue,
-					}
-
-					data, err := output.JSONMarshal()
-					if err != nil {
-						return errors.Wrap(err, "unable to marshal output")
-					}
-
-					err = p.FileSystem.WriteFile(outpath, data, 0755)
+					err := p.FileSystem.WriteFile(outpath, []byte(outputValue), 0755)
```

Comment:
Now that the output data is stored in the claim, and we aren't storing additional data in the output file in porter home, does it make sense to keep storing the output file in ~/.porter/outputs? I'm wondering what value do we get out of having `ApplyBundleOutputs` or if we could/should switch to having all the commands work off of the outputs in the claim in a follow-up?

Reply:
This is writing to the standard path inside the invocation container (`/cnab/app/outputs`).  Indeed, with the changes in this PR, we will no longer be writing/storing outputs on the local user's system (`~/.porter/outputs` will no longer exist)

Reply:
Thanks for the clarification! I did indeed have it all mixed up. 😀